dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -463,6 +465,68 @@ public Response getContent(@Context HttpServletRequest request,
         return responseResource.response(result, null, status);
     }
 
+    /**
+     * This methods receives a Lucene query.
+     * It processes the query looking for special scenarios like structure fields (i.e: stInode, stName) and replace them with valid Content fields
+     * @param luceneQuery
+     * @return luceneQuery
+     */
+    private String processQuery(String luceneQuery) throws DotDataException, DotSecurityException {
+        if (luceneQuery == null) {
+            return null;
+        }
+
+        //Look for stName
+        if (luceneQuery.contains(Contentlet.STRUCTURE_NAME_KEY + ":")) {
+            //Parameter is in the FORMAT  stName:variableName
+            //Replace to FORMAT  ContentType:variableName
+            luceneQuery = luceneQuery.replace(Contentlet.STRUCTURE_NAME_KEY + ":", "ContentType:");
+        }
+
+        //Look for stInode
+        String stInodeKey = Contentlet.STRUCTURE_INODE_KEY + ":";
+        if (luceneQuery.contains(stInodeKey)) {
+            //Parameter is in the FORMAT  stInode:inode
+
+            //Lucene parameters are separated by blankSpace
+            int startIndex = luceneQuery.indexOf(stInodeKey) + stInodeKey.length();
+            int endIndex = luceneQuery.indexOf(" ", startIndex);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Put single-quotes around ' ' to use the faster "indexOf(char)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3027)